package.json

```diff
@@ -31,8 +31,8 @@
     "prebuild": "node generate-netlify-env.js",
     "build": "next build",
     "start": "next start",
-    "test": "npm run test:jest",
-    "test:jest": "jest",
+    "test": "vitest run",
```

Comment:
This will run only once. We appear to use `open-sauced/hot/.github/workflows/development.yml@main` so I didn't want to break the CI/CD pipeline and opted for `test:watch` for local dev.